samples/BikeSharingApp/BikeSharingWeb/Dockerfile

```diff
@@ -1,10 +1,16 @@
-FROM node:lts
-ENV PORT 80
-EXPOSE 80
+# STAGE-1: Build application
+FROM node:lts-alpine AS builder
 
 WORKDIR /app
 COPY package.json .
-RUN npm install
+RUN npm install --only=production
 COPY . .
 
+# STAGE-2: Copy built file into a new docker
```

Comment:
I don't think the second build stage is necessary for this image.  Typically that convention is used to trim away any development-only artifacts from the image, but in this case it looks like they are all being copied over anyway.  Did you have another goal in mind by structuring it this way?

Reply:
true this is sort of just a foot in the door to put in a multi-stage step. In the next PR I'd like to  copy only the `./app/.next` folder but that would need a bit more work and I was hoping to look into it after this goes in.

Reply:
Would you mind removing the second build stage for the purposes of this PR?  (And potentially following up with a fully-implemented "two stage build" PR?)  Sorry to nit-pick!

We have a desire for our `master` branch to guide users into a recommended set of practices, and we fear the current implementation may confuse users.

Reply:
Apologies for the delay, removed the second build stage.

Reply:
Thanks!